src/main/java/core/basesyntax/service/ShopService.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+import java.util.Map;
+
+public interface ShopService {

```

Comment:
According to the checklist, don't begin class or method implementation with an empty line. Please remove the redundant empty lines before the interface declaration.